From 2a74d41325c9e8043a2beec8b957ddf66f57cfd5 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Tue, 18 Dec 2012 02:09:14 -0500
Subject: [PATCH] build/test/install python bindings manually

To support building for multiple python ABIs, we want to skip checks for
python in configure, and build/test/install python bindings manually.
---
 configure.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index 3d5f48b..b467c51 100644
--- a/configure.in
+++ b/configure.in
@@ -748,7 +748,7 @@ PYTHON_INCLUDES=
 PYTHON_SITE_PACKAGES=
 PYTHON_TESTS=
 pythondir=
-if test "$with_python" != "no" ; then
+if false ; then
     if test -x "$with_python/bin/python"
     then
         echo Found python in $with_python/bin/python
@@ -823,7 +823,7 @@ if test "$with_python" != "no" ; then
 else
     PYTHON=
 fi
-AM_CONDITIONAL(WITH_PYTHON, test "$PYTHON_INCLUDES" != "")
+AM_CONDITIONAL(WITH_PYTHON, test "$with_python" != "no")
 if test "$PYTHON_INCLUDES" != ""
 then
     PYTHON_SUBDIR=python
@@ -1429,7 +1429,7 @@ else
     echo "Enabled Schemas/Relax-NG support"
     WITH_SCHEMAS=1
     TEST_SCHEMAS="Schemastests Relaxtests"
-    if test "$PYTHON_INCLUDES" != "" ; then
+    if test "$with_python" != "no" ; then
         PYTHON_TESTS="$PYTHON_TESTS RelaxNGPythonTests SchemasPythonTests"
     fi
     with_regexps=yes
-- 
1.8.0.2

